### PR TITLE
Contact

### PR DESCRIPTION
**Pull request by Jessekay**
- I added a contact form,
- I collected data using Formspree service,
- I used flexbox for laying out elements on the page,
- I implemented button interactions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -152,6 +152,20 @@ <h1 class="lg">Skills</h1>
...
+          <label for="full-name">
+            <input type="text" class="input-text" name="full_name" id="fullname" placeholder="Enter your full name" maxlength="30" required>
+          </label>
+          <label for="email_address">
+            <input type="email" class="input-email" name="email_address" id="email_address" placeholder="Enter your email address" required>
+          </label>
+          <label for="text_message">
+            <textarea name="message" id="text-message" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
+          </label>
+          <button id="contact-btn" class="btn contact-btn" type="submit"> Get in touch</button>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] I would suggest you please stick to the design as much as possible. When I increase the size of responsiveness then the input field is increased but the input field for the text area is not. Please fix it.
- [ ] Also make sure that the background image is placed in the exact area. Please place below then contact form area. See the image below for reference
### Your design
hisss

### Fimga design
Screenshot 2023-02-23 154204

### Your design
his

File: index.html [hunk 2]
```diff
@@ -152,6 +152,20 @@ <h1 class="lg">Skills</h1>
          <hr class="divider2">
         </div>
       </section>
+      <footer class="contact">
+        <h2 class="contact-me"><b>Contact me</b></h2>
+        <p class="more">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/mrgvqyqz">
+          <label for="full-name">
```
Comment (roshan-bajgain) [inline index.html]: - [ ] Form uses the `action` and `method` attributes. The `method` value equals `POST`. Please implement it. Also make sure that the form is send without error

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Jessekay and @Kgomotso196, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @Jessekay, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Highlights

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**